packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -200,6 +200,9 @@ export class LoaderContainerTracker implements IOpProcessingController {
 		this.lastProposalSeqNum = 0;
 		for (const container of this.containers.keys()) {
 			container.close();
+			// Optional chaining here is because containers made with LTS loaders don't have a dispose method or process
+			// and this package is used with various previous versions of Fluid layers in our compat testing.
+			container.dispose?.();
```

Comment:
Talking with Kian briefly on this and he thought doing nothing here for LTS loader should be ok, but I think I was still seeing some leaks there. Probably not going to try and address right now, but might double-check that and add some details to the comment or the related work around turning off mocha's `--exit` flag.

Reply:
I left a comment on AB#7856 pointing to what I know here which isn't definite, but not going to investigate further right now.